devops/README.md

```diff
@@ -1,17 +1,251 @@
+yum install osconfig
+```
+### Remove
+```bash
+yum remove osconfig
+```
+
+## Red Hat Enterprise Linux 9
+### Install
+#### <a name="rhel9-insiders-fast">insiders-fast
+```bash
+yum install yum-utils
+yum-config-manager --add-repo https://packages.microsoft.com/yumrepos/microsoft-rhel9.0-insiders-fast-prod/config.repo
+yum install osconfig
+```
+### Remove
+```bash
+yum remove osconfig
+```
+
+## Rocky Linux 9
+### Install
+#### <a name="rockylinux9-insiders-fast">insiders-fast
+```bash
+yum install yum-utils
+yum-config-manager --add-repo https://packages.microsoft.com/config/rocky/9/insiders-fast.repo
+yum install osconfig
+```
+### Remove
+```bash
+yum remove osconfig
+```
+
+## SUSE Linux Enterprise Server 15
+### Install
+#### <a name="sles15-insiders-fast">insiders-fast
+```bash
+zypper install yum-utils
+zypper ar -f https://packages.microsoft.com/yumrepos/microsoft-sles15-insiders-fast-prod/config.repo
+zypper install osconfig
+```
+### Remove
+```bash
+zypper remove osconfig
+```
+
+## Ubuntu 20.04
+### Install
+#### <a name="ubuntu2004-insiders-fast">insiders-fast
+```bash
+wget https://packages.microsoft.com/config/ubuntu/20.04/insiders-fast.list -O /etc/apt/sources.list.d/packages-microsoft-com_insiders-fast.list
+```
+#### <a name="ubuntu2004-prod">prod
+```bash
+wget https://packages.microsoft.com/config/ubuntu/20.04/prod.list -O /etc/apt/sources.list.d/packages-microsoft-com_prod.list
+```
+```bash
+wget -qO - https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
+apt update
+apt install osconfig
+```
+### Remove
+```bash
+apt remove osconfig
+```
+
+## Ubuntu 22.04
+#### <a name="ubuntu2204-insiders-fast">insiders-fast
+```bash
+wget https://packages.microsoft.com/config/ubuntu/22.04/insiders-fast.list -O /etc/apt/sources.list.d/packages-microsoft-com_insiders-fast.list
+```
+#### <a name="ubuntu2204-prod">prod
+```bash
+wget https://packages.microsoft.com/config/ubuntu/22.04/prod.list -O /etc/apt/sources.list.d/packages-microsoft-com_prod.list
+```
+### Install
+```bash
+wget -qO - https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
+apt update
+apt install osconfig
+```
+### Remove
+```bash
+apt remove osconfig
+```
+
+# DevOps Folder
+This folder contains the artifacts and scripts used by our engineering systems (build, test, package).
 
 - `debian` - Debian packaging scripts
 - `docker/<os>-<arch>` - Build containers for supported operating systems and architectures
   - `amd64`
   - `arm` (armv7)
   - `arm64` (aarch64)
-- `pipeline` - Azure Pipelines definitions
 - `e2e`
   - `cloudtest` - backup of 1ES managed CloudTest image definitions
   - `terraform` - terraform modules for provisioning Azure resources for E2E tests
-
-> See [Actions](https://github.com/Azure/azure-osconfig/actions) for more details and the status of our [workflows](https://github.com/Azure/azure-osconfig/tree/main/.github/workflows).
```

Comment:
Maybe add a badge to the top of this doc for the prod/insiders-fast workflows?

Reply:
already did